### PR TITLE
Add blog post for back office rebranding

### PR DESCRIPTION
**Pull request by tblivet**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add blog post to introduce back office rebranding in PrestaShop v9
| Fixed ticket? | 

post_preview

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hello @tblivet!

This is your first pull request on [prestashop.github.io](https://github.com/PrestaShop/prestashop.github.io/) repository of the PrestaShop project.

Thank you, and welcome to this Open Source community!

File: content/news/posts/2024/prestashop-bo-rebrand.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "PrestaShop Back Office rebranding with PrestaShop 9"
+subtitle: "Rebranded Back Office Themes Now Available in PrestaShop v9"
```
Comment (kpodemski) [inline content/news/posts/2024/prestashop-bo-rebrand.md:4]:
```suggestion
subtitle: "Rebranded Back Office Themes Available in PrestaShop 9"
```

Comment (kpodemski) [inline content/news/posts/2024/prestashop-bo-rebrand.md]:
```suggestion
subtitle: "Rebranded Back Office Themes Available in PrestaShop 9"
```

File: content/news/posts/2024/prestashop-bo-rebrand.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+date:   2024-08-26
+author: [ ThibaultBlivet ]
+icon: icon-leaf
+tags: [ "9.0", Design ]
+---
+
+By the end of 2023, PrestaShop unveiled a new brand identity to strengthen its position, enhance credibility, and build trust. The PrestaShop back office is one of the last assets to receive this rebranding, and the release of PrestaShop 9 provided the perfect opportunity for this update. The rebranding involved significant changes to the PrestaShop back office, focusing on its two themes.
```
Comment (kpodemski) [inline content/news/posts/2024/prestashop-bo-rebrand.md]:
```suggestion
By the end of 2023, PrestaShop unveiled a new brand identity to strengthen its position, enhance credibility, and build trust. [We already announced]({{< relref "/news/posts/2023/new-project-branding" >}}) the new branding for the project, and the PrestaShop back office is one of the last assets to follow this change. The release of PrestaShop 9 provided the perfect opportunity for this update. The rebranding involved significant changes to the PrestaShop back office, focusing on its two themes.
```

File: content/news/posts/2024/prestashop-bo-rebrand.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+![Back Office rebrand](/assets/images/2024/08/bo_rebrand.png)
+
+## Back Office Rebrand
```
Comment (kpodemski) [inline content/news/posts/2024/prestashop-bo-rebrand.md]:
```suggestion
## Back office rebranding
```

File: content/news/posts/2024/prestashop-bo-rebrand.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+### Anatomy of the PrestaShop Back Office
```
Comment (kpodemski) [inline content/news/posts/2024/prestashop-bo-rebrand.md]:
```suggestion
### Anatomy of the PrestaShop back office
```

File: content/news/posts/2024/prestashop-bo-rebrand.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+#### Theme: new-theme
+- Used for Symfony controllers
+- Based on **Bootstrap 4**
+- Utilizes the **PrestaShop UI Kit**
+
+While the **new-theme** already incorporated the PrestaShop UI Kit, the **default** theme did not. To unify the appearance across both themes, we introduced the `_root.scss` file to the **default** theme in PrestaShop v9, ensuring both themes now share the same design foundation. This update involved a substantial refactoring effort, particularly around the use of CSS variables, to standardize and centralize design elements.
+
+## Need for Refactoring
+
+Updating both themes was essential to provide a consistent user experience throughout the back office. This harmonization effort focused on integrating CSS variables from the UI Kit, offering a centralized method to define design elements like colors, backgrounds, and shadows. The `_root.scss` file serves as a single source of truth for these variables, enabling efficient application of styles across both themes.
```
Comment (kpodemski) [inline content/news/posts/2024/prestashop-bo-rebrand.md]:
```suggestion
Updating both themes was essential to providing a consistent user experience throughout the back office. This harmonization effort focused on integrating CSS variables from the UI Kit, offering a centralized method to define design elements like colors, backgrounds, and shadows. The `_root.scss` file serves as a single source of truth for these variables, enabling efficient application style across both themes.
```

File: content/news/posts/2024/prestashop-bo-rebrand.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+## Developer Tool for the Default Theme
```
Comment (kpodemski) [inline content/news/posts/2024/prestashop-bo-rebrand.md]:
```suggestion
## Developer tool for the default theme
```

File: content/news/posts/2024/prestashop-bo-rebrand.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+These updates were crucial for ensuring that the UI Kit components adhere to the new brand guidelines, thus providing a cohesive look across the entire back office. By updating the UI Kit components to use these CSS variables, we ensure that the rebranded design is applied consistently, making future updates easier to manage and more scalable.
+
+- [Exposed CSS variables](https://github.com/PrestaShop/prestashop-ui-kit/blob/1c255d96d79c69e2d3e0dd1712f76379941c06bb/scss/_root.scss#L122)
+- [PrestaShop UI Kit (Storybook)](https://build.prestashop-project.org/prestashop-ui-kit/)
+
+## The Role of CSS Variables
+
+CSS variables play a critical role in both the rebranding and the long-term maintainability of PrestaShop’s back office. They not only facilitate current design needs but also lay the foundation for future enhancements.
+
+### Why Use CSS Variables?
```
Comment (kpodemski) [inline content/news/posts/2024/prestashop-bo-rebrand.md]:
```suggestion
### Why CSS Variables?
```

File: content/news/posts/2024/prestashop-bo-rebrand.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "PrestaShop Back Office rebranding with PrestaShop 9"
+subtitle: "Rebranded Back Office Themes Now Available in PrestaShop v9"
+date:   2024-08-26
+author: [ ThibaultBlivet ]
+icon: icon-leaf
+tags: [ "9.0", Design ]
```
Comment (kpodemski) [inline content/news/posts/2024/prestashop-bo-rebrand.md]:
```suggestion
tags: [ "9.0", design, ux ]
```

**Review by kpodemski — CHANGES_REQUESTED** [review]
Great job, @tblivet !

I'm adding some suggestions to it, and we can publish it tomorrow.

**Comment by tblivet** [comment]
Hello @kpodemski, I have just added your suggestions. Everything should be good now 👍

File: content/news/posts/2024/prestashop-bo-rebrand.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "PrestaShop Back Office rebranding with PrestaShop 9"
+subtitle: "Rebranded Back Office Themes Available in PrestaShop 9"
```
Comment (coffrep) [inline content/news/posts/2024/prestashop-bo-rebrand.md]:
```suggestion
subtitle: "PrestaShop 9 Embarks The Newly Rebranded Back Office Themes"
```

File: content/news/posts/2024/prestashop-bo-rebrand.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+date:   2024-08-26
```
Comment (coffrep) [inline content/news/posts/2024/prestashop-bo-rebrand.md]:
```suggestion
date:   2024-09-17
```

**Review by coffrep — APPROVED** [review]
Hi @tblivet, great piece of content, very well written. I've just added two minor edits, of course we should change the date to when this is posted. All good for me, thanks so much for this great Build post! Approved with edits on my end.

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thank you @tblivet @coffrep :)

**Pull request merged**